package.json

```diff
@@ -70,12 +70,20 @@
     "@storybook/addon-actions": "4.1.18",
     "@storybook/addon-storyshots": "4.1.18",
     "@storybook/react-native": "4.1.18",
+    "@tsconfig/react-native": "^2.0.3",
```

Comment:
Note: we've been using pinned versions on package.json. You might want to adjust these additions

Reply:
Fixed.